### PR TITLE
 MOBILSDK-2332: Introduced Feedback Sample

### PR DESCRIPTION
**Pull request by awangvan**

### CODE REVIEW

File: Podfile [hunk 1]
```diff
@@ -1,8 +1,11 @@
+source 'https://github.com/CocoaPods/Specs.git'
+source 'https://github.com/foreseecode/podspecs'
+
```
Comment (awangvan) [inline Podfile]: We don't want to merge this to master, right?

File: Podfile [hunk 2]
```diff
@@ -1,8 +1,11 @@
...
 platform :ios, '10.0'
 workspace 'ForeSeeSamples.xcworkspace'
 
 def foresee_pods
-    pod 'ForeSee', '5.1.0'
+    pod 'ForeSee-test', '5.2.0-RC19'
```
Comment (awangvan) [inline Podfile]: We don't want to merge this to master, right?

File: Podfile [hunk 3]
```diff
@@ -36,3 +39,10 @@ target "LocalNotificationSample" do
     project 'LocalNotificationSample/LocalNotificationSample.xcodeproj'
     foresee_pods
 end
+
+target "FeedbackSample" do
+    project 'FeedbackSample/FeedbackSample.xcodeproj'
+    foresee_pods
+    # include the optional Feedback pod
+    pod 'ForeSee-test/ForeSeeFeedback', '5.2.0-RC19'
```
Comment (awangvan) [inline Podfile]: Perhaps we should create a ticket to make sure ForeSee-test will not be used in the master branch?

File: FeedbackSample/Bridging-Header.h [hunk 4]
```diff
@@ -0,0 +1,10 @@
+//
+//  Use this file to import your target's public headers that you would like to expose to Swift.
+//
+
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+#import <ForeSee/ForeSee.h>
+#import <ForeSeeFeedback/ForeSeeFeedback-Swift.h>
```
Comment (awangvan) [inline FeedbackSample/Bridging-Header.h]: I wonder how this should be done in an objc project?

File: Podfile (hunk 1 above)
Comment (wayneburkett) [reply Podfile]: Right. We'll need to do a final update to point back to the release pod and then hold this ticket waiting for the release.

File: Podfile (hunk 2 above)
Comment (wayneburkett) [reply Podfile]: Yep. See above.

File: FeedbackSample/Bridging-Header.h (hunk 4 above)
Comment (wayneburkett) [reply FeedbackSample/Bridging-Header.h]: I think this is OK for now, unless a client asks for it. Unless @sleeke has another idea? In any event, I guess it should be another ticket?

**Comment by awangvan** [comment]
The samples are now pointing to the alpha pod. Merging to develop

**Pull request merged**